### ISSUE TITLE
Improve EPUB accessibility

### ISSUE THREAD

**Issue by kingjon3377** [kingjon3377/poetry_ebook_utils#2]
Running Daisy ACE on a generated EPUB gives 5 "serious", 7 "moderate", and 1 "minor" accessiblity issues.

The "minor" issue is a missing ARIA role to match the `epub:type` (which is `toc`) on the `nav` element in `nav.xhtml`.

One of the "moderate" issues is that the `schema:accessModeSufficient` metadata is not present in `content.opf`; the remaining "moderate" issues seem to all be that heading levels increase by more than one at a time.

The first "serious" issue is a missing `xml:lang` attribute in `content.opf`; the remainder are missing Schema.org metadata (`schema:accessMode`, `schema: accessibilityFeature`, `schema: accessibilityHazard`, and `schema: accessibilitySummary`) in `content.opf`.

In addition, image `alt` text is almost certainly useless to screen-reader users ...

**Comment by kingjon3377** [comment]
For the `alt` text issue, `ebook_builder.sh` (or an optional sub-script) should parse an illustration-sources TeX file to get image titles and sources.
